docs/specification.md

```diff
@@ -0,0 +1,169 @@
+## General Specification for the project !   
+> This Specification is only for the MVP of this project that need to do the first user journey , consisting of opening a file editing and saving it into the filesystem !    
+--- 
+> This Specification can't be finished now , since m too retarded to give it soo much time !   
+
+### General Directory of the project !   
+``` 
+simple_editor/
+├── Cargo.toml
+├── src
+│   ├── main.rs
+│   ├── commands
+│   │   ├── mod.rs
+│   │   ├── file_commands.rs
```

Comment:
```suggestion
│   │   ├── file_commands.rs
│   │   ├── helper_functions.rs
```

@TheMakarov,

It appears that the temporary specification did not cover the number of functions, structs, and traits allowed per file. To address this, we might need to create a helper_functions.rs file where we can define smaller functions that perform small tricks.

Reply:
i actually thought of em , but i thought why not let them inside one file , respecting a pseudo-SOLID principle !

Reply:
Okay then